src/Microsoft.AspNetCore.DataProtection.Extensions/DataProtectionProvider.cs

```diff
@@ -15,13 +16,37 @@ namespace Microsoft.AspNetCore.DataProtection
     public static class DataProtectionProvider
     {
         /// <summary>
+        /// Creates a <see cref="DataProtectionProvider"/> that store keys in a location based on
+        /// the platform and operating system.
+        /// </summary>
+        /// <param name="applicationName">An identifier that uniquely discriminates this application from all other
+        /// applications on the machine.</param>
+        public static IDataProtectionProvider Create(string applicationName)
```

Comment:
Will we need more overloads of this beyond the `Create(string)`. Will the one with options setup or a cert need to work?